docs/skill-registry-migration.md

```
@@ -0,0 +1,317 @@
+**Consumer** — no auth, no source clone:
+```bash
+curl -fsSL https://raw.githubusercontent.com/Flow-Research/harnessy/main/install.sh | bash -s -- --from-cloud
+```
+Fetches the public lockfile, clones each skill from Artifacts (via `--depth 1 --branch v<version>`), verifies SHA, populates `~/.agents/skills/`.
+
+**Anyone** — verifies their installed skills haven't drifted:
+```bash
+node tools/flow-install/scripts/verify-skills.mjs
+```
+
+## P2P contract (the load-bearing claim)
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)